updater/library/src/updater.rs

```diff
@@ -171,6 +171,7 @@ fn update_internal(config: &ResolvedConfig) -> anyhow::Result<UpdateStatus> {
 
 fn inflate(patch_path: &Path, base_path: &Path, output_path: &Path) -> anyhow::Result<()> {
     info!("Patch is compressed, inflating...");
+    use anyhow::Context;
```

Comment:
This btw, probably just moves to the top level and is used a bunch of other places in this file.

Reply:
(Eventually, not this change.)